concourse/kubectl-sample.txt

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+clusters:
+- cluster:
+  ...
+contexts:
+- context:
+  ...
+current-context: cloud-platforms-test.k8s.integration.dsd.io
```

Comment:
Everything looks fine. I would argue that (even though I know this is a sample file) we should be referencing the non-production cluster as this is where this quick start will run. Not a blocker though.